components/OurCorporates/OurCorporates.js

```diff
@@ -0,0 +1,28 @@
+import { CorporateReachImgNames, CorporateReachLinks } from '../../data';
+import styles from './OurCorporates.module.scss';
+
+const OurCorporates = () => {
+  return (
+    <>
+      <div className={styles.reach}>
+        <h1>Industrial Reach</h1>
+        <div className={styles.reachGroup}>
+          {CorporateReachImgNames.map((name, idx) => {
+            return (
+              <a href={`${CorporateReachLinks[idx]}`}>
+                <img
+                  src={`/static/images/corporates/${name}`}
+                  alt=''
+                  className={styles.reachImg}
+                  key={`alumni_reach_${idx}`}
+                ></img>
+              </a>
+            );
+          })}
+        </div>
+      </div>
+    </>
+  );
+};
+
+export default OurCorporates;
```

Comment:
Why is a new component needed here? It seems like the difference in the components of Corporate Reach and Our Reach is just the titles and the key prefix. If that's the only difference why not just modify the existing component to accept the title and key prefix as parameters?

Reply:
> Why is a new component needed here? It seems like the difference in the components of Corporate Reach and Our Reach is just the titles and the key prefix. If that's the only difference why not just modify the existing component to accept the title and key prefix as parameters?

I though corporates and Universities ko completely different rakhna he so I made different components so that easy to identify in future, images ka bhi different folders banaya he.

Reply:
You have a point, but kind of beats the purpose having Components.

Anyway, having seperate component will be better if someone less familiar will try to edit the website in the future (which usually is the case here).

I think it's not wrong to forego a bit of code quality for better future maintenance.